script_facade/client/client.py

```diff
@@ -122,7 +124,11 @@ def parse_rx_history_response(xml_string, fhir_version, script_version):
 
 def parse_patient_lookup_query(xml_string, script_version):
     # LXML infers encoding from XML metadata
-    root = ET.fromstring(xml_string.encode('utf-8'))
+    try:
+        root = ET.fromstring(xml_string.encode('utf-8'))
+    except ET.XMLSyntaxError as se:
+        current_app.logger.error("Couldn't parse PDMP XML: %s", se)
```

Comment:
Do you need to check if `current_app` is initialized? I'm surprised we don't get some kind of error invoking `parse_patient_lookup_query()` outside of flask.

Reply:
@ivan-c I assume you mean when calling `parse_patient_lookup_query()` from a test without the `client` fixture (which loads a test app context)?

I don't see this as a current problem, as all real use comes from a flask request, right?  If a test hits a `current_app` line without an app context, you will see:
``` RuntimeError: Working outside of application context.
```

Reply:
@pbugni 
Sorry I forgot the fixture added the flask request context
>the client fixture (which loads a test app context)